src/gordon_gcp/plugins/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright 2017 Spotify AB
+# Copyright 2018 Spotify AB
```

Comment:
nitpick, but from a FOSS board perspective, I prefer a date range. If it weren't a date range, it should be 2017.

Reply:
Even if the file in question in from 2018?  This was actually a thing I wanted to ask about.  I will say that if we use ranges, people will 100% never update them.

Reply:
The former, the git diff makes it look like a file that's changed, not a new file. Re: the latter, I'm fine if people forget, that's why I said if it weren't a date range, then 2017.

Reply:
Discussion results: if a file is entirely new, use the current year.  In this case, there was a file there, but it was moved to service/ and this file was created from scratch.

Reply:
Further results: I'll change the dates.